tests/vlan/test_host_vlan.py

```diff
@@ -73,8 +72,10 @@ def setup_host_vlan_intf_mac(duthosts, rand_one_dut_hostname, testbed_params, ve
     wait_until(10, 2, 2, lambda: duthost.get_dut_iface_mac(vlan_intf["attachto"]) == DUT_VLAN_INTF_MAC)
 
     yield
-
-    config_reload(duthost)
+    
+    DUT_HOST_MAC = duthost.get_dut_iface_mac('eth0')
```

Comment:
1. Please use lower case if it is not a constant.
2. MAC address of Vlan interface is not the same as management interface 'eth0' on some platforms. The correct way is to take a note of the original VLAN interface MAC address. Then restore it in teardown section of this fixture.

Reply:
Done, retest passed.